fhir/resource_structure_dev/research_study.json

```diff
@@ -0,0 +1,111 @@
+{
+  "resourceType" : "ResearchStudy",
+  "id" : "example-study-ccpuk",
+  "url" : "",
+  "name" : "CCPUK",
+  "title" : "ISARIC/WHO Clinical Characterisation Protocol UK",
+  "label" : [{
+    "type": {
+      "text": "Study Code"
+    },
```

Comment:
data element 'protocol' can be used to link to the PlanDefinition resource which describes the steps (visits) undertaken as part of the study